packages/mobile/src/dappkit/DappKitAccountScreen.tsx

```diff
@@ -119,42 +114,40 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     justifyContent: 'center',
     marginHorizontal: '15%',
+    marginVertical: '50%',
   },
   header: {
     ...fontStyles.h1,
     alignItems: 'center',
-    paddingBottom: 30,
-  },
-  footer: {
-    flexDirection: 'column',
-    alignItems: 'flex-end',
-    textAlign: 'center',
-  },
-  logo: {
-    marginBottom: 20,
+    paddingBottom: 16,
   },
   share: {
-    ...fontStyles.bodySecondary,
-    fontSize: 13,
-    alignSelf: 'center',
+    ...fontStyles.subSmall,
+    color: colors.gray4,
+    fontSize: 16,
```

Comment:
Once this is switched to `fonts.v2`, we should avoid redefining `fontSize`, unless absolutely unique and necessary.